node-invoker.yaml

```diff
@@ -32,7 +32,7 @@ spec:
 
     For example, from a directory  named 'square' containing a function 'square.js', you can simply type :
 
-        riff {{.Command}} -f square
+        riff {{.Command}} -f .
```

Comment:
`.` is the default, why bother? It would seem easier to remove the either or and only show the latter example.

Reply:
That's fine, not sure where this `-f` example came from to begin with